pkg/lib/oauth/handler/service_token.go

```diff
@@ -32,6 +32,8 @@ type TokenService struct {
 	GenerateToken       TokenGenerator
 	Clock               clock.Clock
 	Users               TokenHandlerUserFacade
+
+	AccessGrantService oauth.AccessGrantService
```

Comment:
Use local interface to generate mock. TokenServiceAccessGrantService

Reply:
It seems we don't need mock here.